src/components/Button/PlayButton.js

```diff
@@ -0,0 +1,15 @@
+import React, { PropTypes } from 'react';
+
+const commonPlayBtnClasses = 'fa fa-play w3 h3 f3 absolute z-1 gray items-center justify-center br-pill pointer card-play-btn'
```

Comment:
@trevordmiller according to @DrShpongle there are some parts of the design that can't be done with straight tachyons (same comment applies to the inline `style` heights)